packages/query-engine/source/middleware/errors/bad-request-error.ts

```diff
@@ -0,0 +1,22 @@
+import { NextFunction, Response, Request } from "express";
+import { constants } from "@hypertool/common";
+
+const { httpStatuses } = constants;
+
+const BadRequestError = (
+    error: any,
+    request: Request,
+    response: Response,
+    next: NextFunction,
+): void => {
+    if (error.code === "ER_BAD_HOST_ERROR") {
+        response.status(httpStatuses.BAD_REQUEST).json({
+            error: "Bad Host Request",
```

Comment:
Is `error` supposed to be a code that the client will understand? If yes, don't include spaces. I'd rather prefer `BAD_HOST_REQUEST`.